composer.json

```diff
@@ -26,11 +26,11 @@
         "yiisoft/yii2": "dev-master",
         "yiisoft/yii2-app-advanced": "dev-master",
         "codeception/verify": "^3.0",
-        "codemix/yii2-localeurls": "^1.7",
         "codeception/module-asserts": ">= 3.0",
         "codeception/module-filesystem": "> 3.0",
         "phpstan/phpstan": "^2",
-        "rector/rector": "^2"
+        "rector/rector": "^2",
```

Comment:
we have this dependency to rector, but I don't see any `rector.php` file
is it a subdependency ?

If we really use rector individually we should either commit the rules file or remove this dependency.

Reply:
Removing it; I thought you added it lol x

Reply:
haha and that's why I don't self-merge PRs